apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -0,0 +1,87 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+.nav {
+	position: fixed;
+	width: 100%;
+}
+
+.navbar {
+	background-image: url("~@/assets/images/lined_paper.svg");
+	background-size: cover;
+	background-position: left bottom;
+	background-repeat: no-repeat;
+	filter: drop-shadow(0px 2.5px 2px rgba(0, 0, 0, 0.4));
+}
+
+.logo {
+	position: absolute;
+	left: 30px;
+	width: 20vw;
+	height: 20vw;
+	text-align: center;
+	transform: rotate(-16.81deg) translateY(-5vw);
+}
+
+.homeNotActive,
+.resourcesNotActive,
+.scheduleNotActive {
+	margin-left: 1vw;
+	margin-right: 1vw;
+	font-size: 2rem;
+	font-weight: bold;
+	color: black;
+	text-decoration: none;
+}
+
+.homeActive {
+	margin-left: 1vw;
+	margin-right: 1vw;
+	font-size: 2rem;
+	font-weight: bold;
+	color: black;
+	text-decoration: none;
+	color: variables.$green;
+}
+
+.resourcesActive {
+	margin-left: 1vw;
+	margin-right: 1vw;
+	font-size: 2rem;
+	font-weight: bold;
+	color: black;
+	text-decoration: none;
+	color: variables.$blue;
+}
```

Comment:
Suggestion: Instead of using the same properties and values for most of these classes, extract the ones that are shared by the active classes all into one ruleset and then leave the differing ones in their own individual rule sets. For example:

```
.rule1, .rule2, .rule3 {
    shared-prop-1: value-1;
    shared-prop-2: value-2;
    ...
}

.rule1 {
    individual-prop-1: value-1;
}

.rule2 {
    individual-prop-2: value-2;
}
...
```